docs/components/arm/_index.md

```diff
@@ -61,7 +61,8 @@ Supported arm models include:
 
 ## Control your arm with Viam's client SDK libraries
 
-To get started using Viam's SDKs to connect to and control your robot, go to your robot's page on [the Viam app](https://app.viam.com), navigate to the **Code Sample** tab, select your preferred programming language, and copy the sample code generated.
+To get started using Viam's SDKs to connect to and control your robot, go to your robot's page on [the Viam app](https://app.viam.com) and navigate to the **Code sample** tab.
+Then, select your preferred programming language, toggle **Include secret** to display your robot's location secret in the code, and copy the sample code generated.
```

Comment:
Sorry - I think we can still improve this further. I don't really like the flow we are suggesting to the user because following it makes it easy for users to introduce security risks in their code if they push it to GH. It may be worth turning this into an include because we will probably change it further as the UI gets changed. How about something like this:

```suggestion
Then, select your preferred programming language, and copy the sample code generated.

By default, the copied sample code does not include your **location secret**.
We strongly recommend adding your location secret as an environment variable and importing is as needed.

You can obtain the [location secret]() by toggleing **Include secret** to display your robot's location secret in the code or on the [Location's page](/manage/fleet/locations/#share-a-location-using-location-secret-keys).

{{% snippet "secret-share.md" %}}
```